### ISSUE TITLE
Improve CanJS.com’s usability

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#4117]
__TLDR:__ we’ve identified multiple issues with CanJS.com’s usability and we propose addressing most/all of the issues below.

## Motivation

We’ve conducted multiple usability studies that have given us insight into how people learn and use CanJS. From each study, we’ve determined things we can do to improve the site’s ease of use.

By addressing these issues, we think both new and existing users will have a better experience on our website.

## The issues

As part of this proposal, we would address most or all of the issues below:

- [Improve the home page](https://github.com/canjs/canjs/issues/4055)
- [Improve the quality of the graphics on the Tech Overview page](https://github.com/canjs/canjs/issues/4056)
- [Add a Comparison to Other Frameworks page](https://github.com/canjs/canjs/issues/2679)
- [Fix navigating back after clicking a link to a specific section](https://github.com/canjs/bit-docs-html-canjs/issues/449)
- [Outline should not include headings within signatures](https://github.com/canjs/bit-docs-html-canjs/issues/455)
- [Make "Chat" and "Forum" buttons more clearly a means of immediate support](https://github.com/canjs/canjs/issues/4096)

## SEO

Additionally, we’ve identified some issues with our SEO. These are important to fix as part of the website’s usability because using a search engine is a common way for people to find content on the site.

- [Prevent pages with @hide from being indexed by search engines](https://github.com/bit-docs/bit-docs-generate-html/issues/50)
- [Generate better page titles](https://github.com/bit-docs/bit-docs-generate-html/issues/44)
- [Give each website page a unique description](https://github.com/donejs/donejs/issues/1094)
- [Fix the Google sitelinks under canjs.com](https://github.com/canjs/canjs/issues/4074)
- [Other CanJS.com issues](https://github.com/canjs/canjs/issues/4079)

This proposal is focused on CanJS.com, but would have positive improvements on DoneJS.com and StealJS.com because some of the code is shared.

As a stretch goal, we could tackle issues on those sites as well:

- [Other DoneJS.com issues](https://github.com/donejs/donejs/issues/1095)
- [Other StealJS.com issues](https://github.com/stealjs/stealjs/issues/99)
